test/vanilla/acceptanceTests.ts

```diff
@@ -489,7 +510,13 @@ describe('typescript', function () {
       it('should get invalid byte value', function (done) {
         testClient.byteModel.getInvalid(function (error, result) {
           should.not.exist(error);
-          assert.deepEqual(result, new Buffer(':::SWAGGER::::', 'base64'));
+          // Base64 decoding of the string ':::SWAGGER::::'
+          const expected = new Uint8Array([73, 96, 6, 24, 68]);
```

Comment:
This smells funny--will want to check on what the real intent of this test is and maybe adjust accordingly